### ISSUE TITLE
[v0.8-beta4] LanguageServer cannot include files with names that contain uppercase characters on Windows

### ISSUE THREAD

**Issue by helgee** [JuliaEditorSupport/julia-vscode#332]
MWE:

**run-test.jl**

```julia
include("test1.jl")
include("Test2.jl") # [LanguageServer.LSDiagnostic] Could not include Test2.jl
println(TEST1)
println(TEST2) # [LanguageServer.LSDiagnostic] Possible use of undeclared variable TEST2
```

**test1.jl**

```julia
const TEST1 = 1
```

**Test2.jl**

```julia
const TEST2 = 2
```

**Comment by ZacLN** [comment]
@davidanthoff  can you replicate this?

**Comment by davidanthoff** [comment]
Sort of, I actually get an error that it can't include the first line, and then an error that ``TEST1`` is not defined... So just the other way around from the original error.

I know that there is an issue with path names and capitalization and includes somewhere in LS, I came across that in the past. I started investigating, then thought the right way to handle this would be to use https://github.com/rofinn/FilePaths.jl for all paths instead of ``String``s, hoping that this would just handle all the platform dependencies. But, it turns out that FilePaths isn't really tested/deved on Windows, so while nice in theory it won't help in practice...
